src/dumbo/src/tcp/connection.rs

```diff
@@ -1161,6 +1161,7 @@ pub(crate) mod tests {
             }
 
             // Calling write_next_segment again should result in a ConnectionReset error.
+
```

Comment:
This blank line here is a bit jarring for some ppl :(

Reply:
Ups. Removed it.